symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/command/HelpCommand.java

```diff
@@ -27,7 +25,7 @@ public class HelpCommand extends SlashCommand {
   private final MessageService messageService;
 
   public HelpCommand(@Nonnull ActivityRegistry activityRegistry, @Nonnull MessageService messageService) {
-    super(HELP_COMMAND, true, c -> {}, DEFAULT_DESCRIPTION);
+    super(HELP_COMMAND, true, false, c -> {}, DEFAULT_DESCRIPTION);
```

Comment:
this change is not necessary, right ?

Reply:
it is. 
The default constructor of SlashCommand has a new field now.